plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -52,7 +68,95 @@ export default function Content(): JSX.Element {
 		} ) );
 	};
 
+	const loadMoreProducts = useCallback( () => {
+		setIsLoadingMore( true );
+		const params = new URLSearchParams();
+		const abortController = new AbortController();
+
+		if ( query.category && query.category !== '_all' ) {
+			params.append( 'category', query.category );
+		}
```

Comment:
For some reason `query.category` is undefined when I'm on the themes tab, and instead of searching for themes it searches for extensions

https://github.com/user-attachments/assets/6faeced0-1ab9-4d22-a872-9bc0a1eb7c3c

Reply:
Possibly related issue: the product state is maintained when I switch tabs.

Scenario:
1. Open page
2. Click load more in themes (page -> 2)
3. Switch to extensions tab, click load more in extensions (page -> 3)

Reply:
Ah, I see what's happening. I'm checking for `query.category`, but when switching between the tabs `query.tab` is changed, and a category query param is only added if you click on a category filter.

Reply:
Should be fixed by https://github.com/woocommerce/woocommerce/pull/51434/commits/a26a03d7e68fc32525e2e1029e14d5319d58e8b7

Reply:
Works. Thanks!